### ISSUE TITLE
This looks cool

### ISSUE THREAD

**Issue by zeke** [getstation/repackage-chrome-extension-for-electron#1]
😎

**Comment by shopglobal** [comment]
Yeah but so far, it just copied my chrome extension into a new directory.. What next, no docs no usage examples, and this is lacking everything needed to "make use of" at this stage.... Are these methods deprecated or is this package not maintained?

**Comment by shopglobal** [comment]
How do you use it? Thanks!
This did nothing but copy directories and left me wondering what exactly happened?????
```
$ npm install @getstation/repackage-chrome-extension-for-electron
```
```
$ repackage-chrome-extension <extension directory> <target directory>
```

**Comment by zeke** [comment]
cc @alexstrat in case they're not watching the repo

**Comment by alexstrat** [comment]
@shopglobal I understand your struggle!

This repository was originally made to work around https://github.com/electron/electron/pull/9494, that was split in https://github.com/electron/electron/pull/10076, https://github.com/electron/electron/pull/11131 and https://github.com/electron/electron/pull/10075 which are all merged: so you probably don't need this repository.

Why do you think you might need this?

**Comment by bitrix1** [comment]
I thought that this would help convert the extension. But there are no connection examples, for example

> "chromium-args": "--load-extension=Extensions\\iifchhfnnmpdbibifmljnfjhpififfog\\1.2.7_0,Extensions\\fmkadmapgofadopljbjfkapdkoienihi\\3.2.4_0",

is ignored by the electron.
